src/avatar.jsx

```diff
@@ -28,75 +28,57 @@ let Avatar = React.createClass({
   },
 
   render() {
-
     let {
-      icon,
       backgroundColor,
       color,
+      icon,
       size,
       src,
       style,
       ...other,
     } = this.props;
 
-    const boxShadow = (style && style.boxShadow) ? style.boxShadow : '0 0 1px 0 rgba(0, 0, 0, 0.2) inset';
-    const borderRadius = (style && style.borderRadius) ? style.borderRadius : '50%';
-
     let styles = {
       root: {
         height: size,
         width: size,
         userSelect: 'none',
         backgroundColor: backgroundColor,
```

Comment:
@oliviertassinari `backgroundColor` should not be set for image avatars as well.

With `backgroundColor`:
![image](https://cloud.githubusercontent.com/assets/2007468/8914021/50416c2a-3462-11e5-8232-eff1a9c3d271.png)

Without `backgroundColor`:
![image](https://cloud.githubusercontent.com/assets/2007468/8914040/6be30df8-3462-11e5-9436-c90100b87638.png)